packages/flutterfire_cli/lib/src/firebase/firebase_android_writes.dart

```diff
@@ -7,9 +7,9 @@ import '../common/utils.dart';
 import '../flutter_app.dart';
 import 'firebase_options.dart';
 
-// https://regex101.com/r/zIMgBI/1
+// https://regex101.com/r/Lj93lx/1
 final _androidBuildGradleRegex = RegExp(
-  r'''(?:(?<indentation>^[\s]*?)classpath\s?['"]{1}com\.android\.tools\.build:gradle:.*?['"]{1}\s*?$)''',
```

Comment:
Are people's current Flutter project maybe generated with an older version will fail? Or is the next regexp working for old projects as well?

Reply:
Great thanks :D